bin/yaml_to_json.py

```diff
@@ -0,0 +1,275 @@
+import yaml, json
+from chalice import BadRequestError
```

Comment:
remove this requirement since, it won't run with the requirements.txt currently on the project since it does not include it.

Reply:
Fixed.